code-security/admin_guide/scan-monitor/sbom-generation.adoc

```diff
@@ -1,83 +1,112 @@
 
-* *Open-source packages* - findings from scans of open source dependencies
-* *Infrastructure as Code* - findings from IaC scans
-* *Images* - packages found in images built or referenced in the repository
+Generating an SBOM in a CycloneDX format will generate a single file with sections like:
 
-Selecting *All* will either generate three separate files for each resource type if you use CSV format, or a merged file for all resource types if you use CycloneDX.
+* Metadata
+* Components
+* Dependencies
+* Vulnerabilities
+
+image::sbom-4.png[width=600]
 
-image::sbom-3.png[width=600]
 
 [.task]
 
-=== Generating an SBOM from a CI/CD Pipeline
+=== Generate SBOM on Prisma Cloud console
 
-You can generate an SBOM from your CI/CD pipeline by adding `-o cyclonedx` or `-o csv` to the end of your CLI command (for a selected directory or file). For example, the command `checkov -d . --bc-api-key PRISMA_ACCESS_KEY::PRISMA_SECRET_KEY -o cyclonedx` will scan a directory and output the findings in CycloneDX XML format.
+You can generate an SBOM report on the Prisma Cloud console in two ways.
 
-[.task]
+* Supply Chain
++
+On *Code Security > https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-code-security/scan-monitor/supply-chain-security[Supply Chain]* generate an SBOM report to get insights into the relationship between the software and supply chain.
++
+image::sbom-1.png[width=600]
 
-=== Supported SBOM Formats
+* Development Pipelines
++
+On *Code Security > https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-code-security/scan-monitor/development-pipelines [Development Pipelines]* generate SBOM to get insights into the open source packages used during the build, and the cascading risk affects that may be on runtime packages.
++
+image::sbom-2.png[width=600]
 
-Prisma Cloud currently supports the export of SBOM reports in two standardized formats: CSV and CyclondeDX.
 
-* CSV is a comma-separated format that displays a line per vulnerability or misconfiguration and a line per resource or package without issues, sorted by groups of open source packages, container images, IaC components, and cloud infrastructure components.
-* CycloneDX output follows the scheme outlined for XML version 1.4.
+[.procedure]
 
-==== SBOM in CSV Format
 
-Generating an SBOM in CSV format from the CI/CD pipeline will generate three separate files:
+. Select a repository to generate a report.
 
-* YYYYMMDD-HHMMSS_iac.csv
-* YYYYMMDD-HHMMSS_container_images.csv
-* YYYYMMDD-HHMMSS_oss_packages
+. Select *Output*.
++
+You can choose between CSV or CycloneDX.
 
-Where YYYYMMDD-HHMMSS is the timestamp for when the CSV was generated, for example: 20220801-221830_iac.csv. This file displays the following data for each IaC finding:
+. Select *Materials*. 
++
+You can choose the type of resources to include in the SBOM.
++
+* *All*: This will generate three separate files of CSV, CycloneDX and a merge file of all resource types.
```

Comment:
This isn't accurate

`* *All*: This will generate three separate files for each of the resource types. If there are no resources of that type, a blank file is generated.`